assert_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+
+package gosnowflake
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func assertNilF(t *testing.T, actual any, descriptions ...string) {
+	errMsg := validateNil(actual, descriptions...)
+	if errMsg != "" {
+		t.Fatal(errMsg)
+	}
+}
+
+func assertNilE(t *testing.T, actual any, descriptions ...string) {
+	errMsg := validateNil(actual, descriptions...)
+	if errMsg != "" {
+		t.Error(errMsg)
+	}
+}
+
+func assertNotNilF(t *testing.T, actual any, descriptions ...string) {
+	errMsg := validateNotNil(actual, descriptions...)
+	if errMsg != "" {
+		t.Fatal(errMsg)
+	}
+}
+
+func assertNotNilE(t *testing.T, actual any, descriptions ...string) {
+	errMsg := validateNotNil(actual, descriptions...)
+	if errMsg != "" {
+		t.Error(errMsg)
+	}
+}
+
+func assertEqualF(t *testing.T, actual string, expected string, descriptions ...string) {
```

Comment:
Can we have `actual` and `expected` as `any`?

Reply:
done